src/fable-library-py/fable_library/date.py

```diff
@@ -197,6 +238,10 @@ def utc_now() -> datetime:
     return datetime.utcnow()
 
 
+def today() -> datetime:
+    return datetime.replace(now(), hour=0, minute=0, second=0, microsecond=0)
+
+
 def to_local_time(date: datetime) -> datetime:
     return date.astimezone()
```

Comment:
I'm worried about this one compared to #3691 since I think you will get problems when localtime is the same timezone as utc (+00). This is why I also removed the tzinfo in my PR

Reply:
I believe the range of the comment is off ?

But yes, there is something to think about here